lib/raxx/context/server_context.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Raxx.Context.ServerContext do
```

Comment:
I don't think I like either of the renames. I think `Connection` nor `Channel` give misleading ideas as to what the module would do or what information it would contain. Both "connection" and "channel" are things that you could potentially send something over, where this is just some context information shared between your modules.

Additionally, there's already a `Channel` and `Connection` in Ace. I know it's a very separate project, but I think they represent more suitable concepts there.

While I think `Raxx.Context` is a good name for all this, I'm still not sure about `Raxx.ServerContext`. Maybe something like `Raxx.Context.ServerProperties` would be better?

I'd also be fine with removing ServerContext altogether for now until there's an actual need for, and allowing the HTTP servers put the info wherever they see fit, and maybe standardising all of it later. Maybe trying to do it now is a bit premature

Reply:
I could go for leaving `Raxx.Context` and dropping `ServerContext` for now.

Reply:
Dropped it 👍